packages/zoe/test/unitTests/contracts/escrowToVote.js

```diff
@@ -0,0 +1,118 @@
+/* global harden */
+// @ts-check
+
+// Eventually will be importable from '@agoric/zoe-contract-support'
+import { assert } from '@agoric/assert';
+import makeStore from '@agoric/store';
+import { makeZoeHelpers } from '../../../src/contractSupport';
+
+/**
+ * Implement coin voting. Give a voting capability when a payment is
+ * escrowed.
```

Comment:
If this is going to be added to our library of contracts, it should have a more complete comment.

Reply:
Very good point

Reply:
Just to be clear, it's been added to the test directory, so not as a reusable component yet, but as an example that we can point to if need be.